Cargo.lock

```diff
@@ -930,9 +935,9 @@ dependencies = [
 name = "ra_cli"
 version = "0.1.0"
 dependencies = [
- "clap 2.33.0 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
looks like something else is still using `clap` in the crate graph?

Reply:
oops, you're totally right, it's `ra_tools`